README.md

```diff
@@ -72,7 +84,20 @@ Add-on modules provides additional functionality to the distribution.
 
 ## Get started with KFD
 
-To get started with KFD, please head to the [documentation site](https://docs.kubernetesfury.com/docs/distribution/#%EF%B8%8F-how-do-i-get-started)
+If you are new to KFD and want to get started, please head to the [documentation site](https://docs.kubernetesfury.com/docs/distribution/#%EF%B8%8F-how-do-i-get-started)
+
+If you are an advanced user, you can get started by using [`furyctl`][furyctl] to download a `Furyfile.yml` and a
+`kustomization.yaml` template file:
+
+```bash
+furyctl distribution template --version v1.23.1
+```
+
+Edit the files accordingly to your needs and then download the modules by running:
+
+```bash
+furyctl distribution download
+```
 
```

Comment:
I think we are missing a call to action here at the end, I suggest adding something like:

```suggestion

You can find now the modules manifests in the `vendor` folder and start working with them, refer to each module's documentation for further details.

```